### PR TITLE
[stable/airflow] fix(requirements): updated versions

### LINKED ISSUE #19585: [stable/airflow] Helm install airflow error
**Version of Helm and Kubernetes**:
helm: 3.0.1
kubernetes: 1.16.2

**What happened**:
Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1", unable to recognize "": no matches for kind "StatefulSet" in version "apps/v1beta2"]

**What you expected to happen**:
Deployment

**How to reproduce it** (as minimally and precisely as possible):
helm install airflow --namespace "airflow" stable/airflow

### PR DESCRIPTION
**Pull request by deblasis**
Fixes #19585 

Fixes installation errors on recent K8S versions

Signed-off-by: Alessandro De Blasis <alex@deblasis.net>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by deblasis** [comment]
Not sure why circleci reports:

> Error: requirements.lock is out of sync with requirements.yaml

`helm lint` passes successfully:
`helm dependency build .` passes successfully too after downloading the updated dependencies

What am I doing wrong? :)

File: stable/airflow/requirements.lock [hunk 1]
```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.13.1
+  version: 8.1.4
```
Comment (gsemet) [inline stable/airflow/requirements.lock]: Can you remove this file, their is no reason to commit the lock file actually !

Comment (deblasis) [reply stable/airflow/requirements.lock]: I see! Sure

**Comment by dlstadther** [comment]
@deblasis 
There were config key changes to stable/postgresql past version 2.0.0. Without update to airflow's values.yaml for postgres (where enabled), it will not run correctly. Check out [this pr](https://github.com/helm/charts/pull/8004/files) for those changes. I'm still working through them myself.

**Comment by deblasis** [comment]
Thanks for the heads up @dlstadther, I will have a look and update the PR accordingly

**Comment by gsemet** [comment]
ok for me. Can you check the linter and add an small description in the README about what user should do in their chart to update to version 6, please?

**Comment by deblasis** [comment]
> ok for me. Can you check the linter and add an small description in the README about what user should do in their chart to update to version 6, please?

That should be it I guess

**Comment by gsemet** [comment]
perfect !

thanks a lot

/lgtm

**Comment by deblasis** [comment]
Hi there, tests are green ✅, we need a volunteer for a **lgtm** 😁

**Pull request merged**